sdk/synapse/Azure.Analytics.Synapse.Artifacts/tests/samples/Sample1_ExecutePipeline.cs

```diff
@@ -16,12 +16,21 @@ namespace Azure.Analytics.Synapse.Samples
     /// </summary>
     public partial class ExecutePipelines
     {
+        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/17455")]
         [Test]
         public async Task CreateAndRunPipeline()
         {
             const string PipelineName = "Test-Pipeline";
```

Comment:
Nit - do these need to be unique across an account?    I could see multi-platform runs getting grumpy if so.

Reply:
StartCreateOrUpdatePipelineAsync will update the existing one if it exists.

I'm rewriting this test when I convert to markdown, as I figured out why I needed to [Ignore].